core/src/main/java/org/apache/carbondata/core/datamap/DataMapDistributable.java

```diff
@@ -34,7 +34,7 @@
 
   private String tablePath;
 
-  private Segment segment;
+  protected Segment segment;
```

Comment:
Why is it needed, please write in comment

Reply:
reverted